wscript

```diff
@@ -412,9 +412,9 @@ iconv support use --disable-iconv.",
 ]
 
 ffmpeg_pkg_config_checks = [
-    'libavutil',     '>= 56.6.100',
-    'libavcodec',    '>= 58.7.100',
-    'libavformat',   '>= 58.0.102',
+    'libavutil',     '>= 56.7.100',
+    'libavcodec',    '>= 58.10.100',
+    'libavformat',   '>= 58.9.100',
```

Comment:
Is this within the scope of an interface-changes entry? I guess we have to be prepared for this to be the beginning of the end of our libav support. Or are we really banking on libav adopting this before our next release?